packages/manager/modules/billing/src/orders/billing-orders-main.controller.js

```diff
@@ -0,0 +1,17 @@
+import get from 'lodash/get';
+
+export default class BillingOrdersMainCtrl {
+  /* @ngInject */
+  constructor(constants, currentUser, coreConfig, featuresAvailabilities) {
+    this.featuresAvailabilities = featuresAvailabilities;
+    this.coreConfig = coreConfig;
+    this.billingGuideUrl = get(
+      constants.urls[currentUser.ovhSubsidiary],
+      'guides.billing',
+    );
+    this.purchaseOrdersGuideUrl = get(
+      constants.urls[currentUser.ovhSubsidiary],
+      'guides.purchaseOrders',
+    );
```

Comment:
if duable, we have to use native language functions.
```suggestion
    const CUSTOMER_URLS = constants.urls[currentUser.ovhSubsidiary];
    
    this.billingGuideUrl = CUSTOMER_URLS['guides']['billing'];
    this.purchaseOrdersGuideUrl = CUSTOMER_URLS['guides']['purchaseOrders'];
```